lib/net.c

```diff
@@ -1330,6 +1330,8 @@ gatekeeper_init_network(struct net_config *net_conf)
 	if (net_conf == NULL)
 		return -1;
 
+	net_conf->log_type = gatekeeper_logtype;
+
```

Comment:
Shouldn't we be setting the log level of the Gatekeeper type to `net_conf->log_level` here?

Reply:
Yes, thanks. The latest rebase has this in it.